index.html

```diff
@@ -116,25 +116,33 @@
         profile in an unambiguous way. This mechanism can be
         used to define additional profiles.
       </li>
-      <li>In addition it defines a <strong><a>Core
+      <li>It define a first base <strong><a>Core
```

Comment:
please clarify what you mean with "a first base"

Reply:
What you call as "Core" is the first introduced Profile in the specification, right?  So it is the "first base". 
However, I have no hard opinion about that. We can change this back if you want.

Reply:
I removed the 'base' term in the text